### PR TITLE
Move some Config tabs to Twig (part 1)

### PR DESCRIPTION
**Pull request by cconard96**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | yes
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | -

### CODE REVIEW

**Review by cedric-anne — APPROVED** [review]
Usage of `showXXX` methods to generate HTML outside Twig template and usage of the `call` Twig function should be avoided as possible, but it can be done in future PRs.

File: templates/pages/setup/general/api_setup.html.twig [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+      copyable: true
+   }) }}
+
+   {{ fields.smallTitle(__('Legacy API')) }}
+   {% if config['enable_api'] %}
+      {% set legacy_doc %}<a href="{{ legacy_doc_url }}">{{ __('API inline Documentation') }}</a>{% endset %}
+      {{ fields.htmlField(
+         '',
+         legacy_doc
+      ) }}
```
Comment (orthagh) [inline templates/pages/setup/general/api_setup.html.twig:67]:
```suggestion
      ) }}
      {{ fields.nullField() }}
```
This to add an empty col fields and align url fields (with above paragraph)

**Review by orthagh — CHANGES_REQUESTED** [review]

File: templates/pages/setup/general/assets_setup.html.twig [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+      0: __('Global'),
+      1: __('By entity'),
+   }, __('Automatic fields (marked by *)')) }}
+   {{ fields.dropdownArrayField('printers_management_restrict', config['printers_management_restrict'], global_management_options, __('Restrict printer management')) }}
+   {{ fields.dropdownArrayField('devices_in_menu', null, item_devices_types, __('Devices displayed in menu'), {
+      multiple: true,
+      size: 3,
+      values: config['devices_in_menu']
+   }) }}
+{% endblock %}
```
[the remark this answers is not in the document]
Comment (cconard96) [reply templates/pages/setup/general/assets_setup.html.twig:63]: Not related to this PR. That section is missing in the main branch. It was changed to be per-entity settings in #14562.

[the remark this answers is not in the document]
Comment (orthagh) [reply templates/pages/setup/general/assets_setup.html.twig:110]: good point

**Review by orthagh — APPROVED** [review]

**Comment by orthagh** [comment]
Fix lint please and we can merge

**Pull request merged**